7.analyze-coverage/get-coverage.py

```diff
@@ -61,15 +60,14 @@ def get_minmax_geneset(row, minmax='min'):
     return pd.Series([z_score, geneset], index=['top_z_score', 'geneset'])
```

Comment:
How does this function know what z_score is?

Reply:
the pandas row stores `z_score` information (and a bunch of other info for the specific geneset)